fix updated package name on pacman/extra (#744)
marlonrichert/zsh-autocomplete: README.md

## Installation & setup
First, install Autocomplete itself. Here are some way to do so:
  * To use only releases (instead of the `main` branch), install `zsh-autocomplete` with a package

>     manager. As of this writing, this package is available through Homebrew, Nix, Plumage,
>     and (as `app-shells/zsh-autocomplete`) Portage, or the package `zsh-completions` using pacman.

Reviewer: ```suggestion
    manager. As of this writing, this package is available through Homebrew, Nix, Plumage, Portage
    (as `app-shells/zsh-autocomplete`), and Pacman (as part of `zsh-completions`).
```
